web/src/index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>Compliance Ops</title>
     <script src="//cdn.auth0.com/js/lock/10.0/lock.min.js"></script>
+    <script src="//auth/js/keycloak.js"></script>
```

Comment:
shoot, I didn't mean to commit that file yet, it's part of a keycloak branch I'm working on and is wrong.